src/motion_input_shaping/README.md

```diff
@@ -52,51 +46,105 @@ To run the example code:
+- [zero_vibration_stream_generator.py](zero_vibration_stream_generator.py) - Contains the implementation of shapers and generate the information required to execute a trajectory through a stream.
+- [damped_vibration.py](damped_vibration.py) - Contains the `DampedVibration` class, which is a basic mathematical implementation of a theoretical damped vibration response curve.
+- [step_response_data.py](step_response_data.py) - Contains the `StepResponseData` class, which is a helper class used for perfoming a move with a Zaber axis while capturing position data via the onboard scope. It is used in the other testing scripts.
+
+### ShapedAxis Class
+
+This class performs shaping by attempting to cancel the impulse from acceleration by changing the timing and magnitude of the deceleration.
+
+This simplified method works well for short moves that are completed in a low number of vibration periods. The vibration cancellation only occurs while decelerating meaning the oscillation will persist during the move and only be reduced when coming to stop. This method becomes less effective with longer moves due to the errors that are accumulating in the time between acceleration and deceleration.
 
-### The ShapedAxis Class
+Performing a move only requires sending a maximum of 3 additional commands so communication overhead is low and will only add a short delay to the start of the move.
 
-Most users will want to reuse the ShapedAxis class in their code. The ShapedAxis class has all the functionality of the Zaber Motion Library [Axis](https://software.zaber.com/motion-library/api/matlab/ascii/axis) class, but adds a few extra commands for performing moves with the zero vibration input shaping algorithm. The general class usage is shown below.
+The general class usage is shown below.
 
 Importing the class (Python):
 
 ```python
 from shaped_axis import ShapedAxis
+from plant import Plant
+```
+
+Initalization (Python):
+
+```python
+plant = Plant(resonant_frequency, damping_ratio)
+shaped_axis_var = ShapedAxis(axis, plant)
+```
+- The `axis` parameter is the [`Axis`](https://software.zaber.com/motion-library/api/py/ascii/axis) or [`Lockstep`](https://software.zaber.com/motion-library/api/py/ascii/lockstep) class instance that the `ShapedAxis` class will perform input shaped moves on.
+- See [here](https://software.zaber.com/motion-library/docs/tutorials/code) for a basic tutorial on how to initialize the `Axis` class.
+- The `plant` parameter is an instance of the `Plant` class which defines the vibration that the input shaper is targeting.
+  - The `Plant` class has `resonant_frequency` and `damping_ratio` properties that define the target vibration frequency in Hz and damping ratio at which the input shaping algorithm will remove vibration.
+
+Class Methods:
```

Comment:
All the `move_absolute_shaped()` methods have been changed to `move_absolute()`, i.e. minus `_shaped()`.  Need to update the documentation for both the ShapedAxis Class and ShapedAxisStream class.

Reply:
Done